modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,175 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+import pytest
+
+import modin
+import modin.experimental.xgboost as xgb
+import modin.pandas as pd
+import numpy as np
+import xgboost
+
+from sklearn.datasets import (
+    load_iris,
+    load_diabetes,
+    load_digits,
+    load_wine,
+    load_breast_cancer,
+)
+from sklearn.metrics import accuracy_score
+from numpy.random import uniform, randint
+
+import multiprocessing as mp
+
+modin.config.NPartitions.put(200)
+
+
+num_cpus = mp.cpu_count()
+
+
+@pytest.mark.parametrize(
+    "num_round",
+    [10, 100, 300],
+    ids=["10", "100", "300"],
+)
+@pytest.mark.parametrize(
+    "num_actors",
+    [1, num_cpus, None, 400],
+    ids=["1", "num_cpus", "None", "400"],
+)
+@pytest.mark.parametrize(
+    "data",
+    [
+        (
+            load_iris(),
+            {
+                "max_depth": 3,
+                "eta": 0.3,
+                "silent": 1,
+                "objective": "multi:softprob",
+                "num_class": 3,
+            },
+        ),
+        (
+            load_diabetes(),
+            {
+                "colsample_bytree": uniform(0.7, 0.3),
+                "gamma": uniform(0, 0.5),
+                "learning_rate": uniform(0.03, 0.3),
+                "max_depth": randint(2, 6),
+                "n_estimators": randint(100, 150),
+                "subsample": uniform(0.6, 0.4),
+            },
```

Comment:
Why do you use random values here? sklearn has an [example](https://scikit-learn.org/stable/auto_examples/ensemble/plot_gradient_boosting_regression.html#sphx-glr-auto-examples-ensemble-plot-gradient-boosting-regression-py) of Gradient Boosting with this dataset, you can use it.

Reply:
`param` for `load_diabets` was changed